gtsam/sfm/sfm.i

```diff
@@ -34,6 +36,9 @@ class SfmData {
   static gtsam::SfmData FromBundlerFile(string filename);
   static gtsam::SfmData FromBalFile(string filename);
 
+  std::vector<gtsam::SfmTrack> tracks;
```

Comment:
Remove this in factor of **const &** for track and camera.
You can also add const& tracks() and cameras() methods.